global/registry.d.ts

```diff
@@ -0,0 +1,9 @@
+declare const _exports: {
```

Comment:
`global` directory shouldn't exist. Move type definition to proper location. See above comment https://github.com/ethereum/remix-project/pull/1344#discussion_r692834298

Reply:
done